src/main/java/org/jenkinsci/plugins/codescene/CodeSceneBuilder.java

```diff
@@ -314,6 +320,21 @@ private CommitRange getRangeFromPreviousSuccessfulCommit(EnvVars env, Commit cur
         }
     }
 
+    private EnvVars getEnvVar(FlowExecution execution) throws ExecutionException, InterruptedException, IOException {
```

Comment:
`getWorkFlowEnvVars` could be a better name (regardless of whether to include `workflow` or not it should be plural).